File: DesafioTecnicoFrame_C#_2020/DesafioTecnico/Model/TODOs.cs
using Newtonsoft.Json;
using System;

namespace DesafioTecnico.Model
{
    public class TODOs
    {
        [JsonProperty("userId")]
        int userId { get; set; }
        [JsonProperty("id")]
        int id { get; set; }
        [JsonProperty("title")]
        string title { get; set; }
        [JsonProperty("completed")]
        bool completed { get; set; }

        public override string ToString()
        {
            return string.Format($"UserId: {userId} \nId: {id} \nTitle: {title} \nCompleted: {completed}");
        }

    }//end class



}//end namespace

File: DesafioTecnicoFrame_Java_2020/src/Model/TODOs.java
package Model;

public class TODOs {

    private Long userId;
    private Long id;
    private String title;
    private Boolean completed;

    public Boolean getCompleted() {
        return completed;
    }

    public void setCompleted(Boolean completed) {
        this.completed = completed;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    @Override
    public String toString() {
        return  "UserId: " + userId +
                "\nId: " + id +
                "\nTitle: " + title +
                "\nCompleted: "+ completed +"\n";
    }//end toString

}//end class
